<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchplace</class>
 <widget class="QWidget" name="searchplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入想查询的地方</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>类别</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>761</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>热度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>评分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>评论数</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>641</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择你想以什么方式排序</string>
   </property>
   <widget class="QRadioButton" name="popularityButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>热度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scoreButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>评分</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
